src/isa/riscv64/instr/rvc/exec.h

```diff
@@ -67,6 +67,14 @@ def_EHelper(c_jalr) {
 #endif
 }
 
+#ifdef CONFIG_SHARE
```

Comment:
Why CONFIG_SHARE?

Reply:
Just for reference model at first, now this PR is modified.